v22.2/hasura-getting-started.md

```diff
@@ -0,0 +1,193 @@
+1. Check **CockroachDB Client to access the databases**, then click **Apply**.
+
+## Create a table in your CockroachDB cluster
+
+1. Connect to your cluster using the SQL client.
+   
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url "{connection string}"
+    ~~~
+
+    Where `{connection string}` is the connection string for your cluster you copied earlier.
+
+1. In the SQL client, create an `accounts` table.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE TABLE IF NOT EXISTS accounts (
+      id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+      balance INT8
+    );
+    ~~~
+
+1. Insert some data into the `accounts` table.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    UPSERT INTO accounts (balance) VALUES (1000), (250);
+    ~~~
+
+This will create two accounts, one with $1000 and another with $250.
+
+## Track the <code>accounts</code> table in the Hasura Cloud Data Manager
+
+1. In the Hasura Cloud Data Manager click `public` under your connection name.
+1. Click the **Track All** button.
+1. Click **Yes** when prompted to expose the table to the GraphQL API.
+
+### Create a GraphQL query in the Hasura Cloud API Explorer
+
+In the Hasura Cloud Explorer, you will see the `accounts` table. You can expand `accounts` to construct a GraphQL query against the `accounts table.
+
+For example, to create a query that finds all accounts with balances greater than or equal to $500:
+
+1. Enter `GetLargeAccounts` as the query name in the Explorer.
+1. Expand `accounts` in the Explorer.
+1. Check `balance` and `id`, then expand `where`, then `balance`.
+1. Check `_gte` to create a "greater than or equal to" where clause for `balance`, then enter `500`.
+
+This creates the following GraphQL query:
+
+{% include_cached copy-clipboard.html %}
+~~~ graphql
+query GetLargeAccounts {
+  accounts(where: {balance: {_gte: "500"}}) {
+    balance
+    id
+  }
+}
+~~~
+
+Click the **Execute Query** button to run the `GetLargeAccounts` GraphQL query. The returned data should look like the following:
+
+~~~ json
+{
+  "data": {
+    "accounts": [
+      {
+        "balance": 1000,
+        "id": "2ed41abf-e831-42df-aff4-f88a6f2b0afc"
+      }
+    ]
+  }
+}
+~~~
+
+<img src="{{ 'images/v22.2/hasura-execute-query.png' | relative_url }}" alt="Executing the GraphQL query in the Hasura Cloud API Explorer" style="border:1px solid #eee;max-width:100%" />
+
+## Next steps
+
+Now that you have connected your cluster to Hasura GraphQL Engine, you can use the Explorer to create GraphQL queries to access and modify data in your cluster's tables. You can then use these queries in your application. For example, you can create [Hasura Actions](https://hasura.io/docs/latest/actions/index/) that use these queries. See the [Hasura documentation](https://hasura.io/docs/latest/index/) for detailed information on integrating and deploying Hasura in your applications.
```

Comment:
Small request :) Can we replace that doc link with this one:
https://hasura.io/docs/latest/index?utm_source=cockroachdb

This is so that Hasura can track where traffic to their doc page is coming from in a better way.

Reply:
Done.